### DISCUSSION TITLE
Auto share contacts when share an account with other users.

### DISCUSSION THREAD

**Discussion by tquoc229** [YetiForceCompany/YetiForceCRM#16938] [How to] [answered]
Hi, 
When I share Account A with a user User X, I want all contacts of Account A also automatically shared with User X.
How can I do this?
Thanks.

**Comment by rskrzypczak** [comment, marked as answer]
You can, for example, try to set permissions based on hierarchy in the role configuration
![image](https://user-images.githubusercontent.com/10221999/222675350-fd34594a-a569-4de1-a5e9-e7acc23238d6.png)

**Reply by tquoc229** [reply]
Awsome, thanks so much @rskrzypczak
